PseudoGantry/lib/PseudoGantry/PseudoAxis.h

```diff
@@ -0,0 +1,45 @@
+#ifndef PSEUDO_AXIS_H
+#define PSEUDO_AXIS_H
+
+#include "Arduino.h"
+
+typedef struct PseudoEncoder
+{
+    uint32_t motor_pulse_pin;
+    volatile uint32_t channel_a_pin;
```

Comment:
Still doesn't need to be volatile right?

Reply:
i dont know why i ever set it to volatile LOL

Reply:
Ya never know, the pins might spontaneously re-arrange on you :)